src/common/utils.js

```diff
@@ -136,3 +136,11 @@ export function markPerformanceIfAble(markName) {
     performance.mark(markName);
   }
 }
+
+// Returns performance timing properties to send in analytics requests
+export function getPerformanceProperties() {
+  return {
+    emitted_at: Date.now() / 1000, // Seconds since epoch, for comparing timing on backend vs. frontend
```

Comment:
I'm not a fan of this but not sure of a better way to compare than using this and `time.time()`